### DISCUSSION TITLE
What documentation system should CLOG have?

### DISCUSSION THREAD

**Discussion by vindarel** [rabbibotton/clog#170] [Ideas]
Hi there,

So currently the CLOG tutorials are in Google Docs.

If you like this theme: https://docsify.js.org/ I can help setting it up. This system doesn't need to generate HTML pages, it transforms the .md ones to HTML on the fly. We need .md doc pages in a docs/ directory, have an index.html that loads docsify (see their doc), and tell GitHub Pages to serve from the docs/. It will read index.html, load Docsify, load the .md files as HTML.

If you want another documentation organization, I can recommend Hugo since it's built in Go (-> fast and easy to install, no dependency hell or breakage on updates), but it's a bit more work.

Or something else, there are plenty.

edit: a cons is that it isn't the best for SEO: no HTML pages, poor SEO.

**Comment by rabbibotton** [comment]
I like markdown in general and pax can generate the current manual also in md. I'll take a look. The alternative is also just to use 3BMD and generate on the fly on the clog website the html.

**Comment by andrej-ozi** [comment]
Hello. If there is interest i can rewrite tutorials from google docs to org-mode. Github formats these quite nicely and conversion to markdown (and vice versa) is pretty straightforward.

**Reply by rabbibotton** [reply]
sounds amazing, maybe start with one and see how it goes and we can coordinate any updates on pics or text.

**Reply by andrej-ozi** [reply]
here is a quick version without any pics. Note i edited some text from the original

https://github.com/andrej-ozi/clog-tutorial

**Reply by rabbibotton** [reply]
looks good, I just load that in to emacs? Is it hard to add pics? that is main reason I went with google docs.

**Reply by andrej-ozi** [reply]
Glad you like it. Its easy to add in pics. I just wanted to show the text format quickly but ill add them. org-mode actually has very good support for advanced literal programming. I will do a quick start guide, starting from vanila emacs, to demonstrate how to get it going
